config/initializers/recaptcha.rb

```diff
@@ -1,9 +0,0 @@
-# config/initializers/recaptcha.rb
```

Comment:
I think you could simply check these files out to `publiclab`'s master branch too, same as the Gemfile.lock.